app/gateway-oss/2.5.x/external-plugins.md

```diff
@@ -55,6 +55,8 @@ in each language:
 plugins = bundled, go-hello, js-hello, py-hello
 ```
 
+**Note:** the start and query commands use a very limited default PATH variable.  In most cases you'll have to specify the full executable path.
```

Comment:
```suggestion
{:.note}
> **Note:** The `start` and `query` commands use a limited default PATH variable. In most cases, you have to specify the full executable path instead.
```

Reply:
if quoted, i guess those would have to use the full variable names, like:

```suggestion
{:.note}
**Note:** the `pluginserver_XXX_start_cmd` and `pluginserver_XXX_query_cmd`
commands use a very limited default `PATH` variable.  In most cases, you'll have to
specify the full executable path.
```

or is it too long?  in any case there's less chance for error.

Reply:
```suggestion
{:.note}
> **Note:** The `pluginserver_XXX_start_cmd` and `pluginserver_XXX_query_cmd` commands use a limited default `PATH` variable. In most cases, you have to specify the full executable path instead.
```
I think you're right, so here you go - one more edit :)